apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -452,7 +464,7 @@ const NftViewer = ({ route }: Props) => {
           {!floorPriceLoading && floorPrice ? (
             <Section
               title={t("nft.viewer.attributes.floorPrice")}
-              value={`${floorPrice} ${ticker}`}
+              value={`${shouldApplyDiscreetMode && discreet ? "***" : floorPrice} ${ticker}`}
```

Comment:
I believe this didn't use the formatting component because the API returns the price already formatted so string concatenation was sufficient? I didn't want to change too much here so kept it the same and applied the ** manually. Any input on this is welcome.

Reply:
your change seems ok to me, at least you're not introducing a regression